src/main/java/ai/knowly/langtorch/schema/io/Metadatas.java

```diff
@@ -0,0 +1,17 @@
+package ai.knowly.langtorch.schema.io;
+
+import java.util.List;
+import java.util.Map;
+
+public class Metadatas {
+
+    private final List<Map<String, String>> values;
```

Comment:
use Multimap instead if there's more than one value associated with the same key

Reply:
oh nice, was not aware of this. It's why before I was thinking we might need to nest a map